homeassistant/components/media_player/apple_tv.py

```diff
@@ -132,6 +146,18 @@ def _has_playing_media_changed(self, new_playing):
         return new_playing.media_type != old_playing.media_type or \
             new_playing.title != old_playing.title
 
+    def turn_on(self):
```

Comment:
Since this only sets simple variables, please turn this into it's async variant `async_turn_off`. Has to be a coroutine. Make sure to also rename `_reset` to be called `_async_reset` and make it a `@callback` (imported from ha.core)